src/runtime/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if(WITH_CUDA_BACKEND)
   target_link_libraries(rt-backend-cuda PRIVATE hipSYCL-rt ${CUDA_LIBRARIES} ${CUDA_DRIVER_LIBRARY})
 
   install(TARGETS rt-backend-cuda
-        EXPORT install_exports
+        #EXPORT install_exports
```

Comment:
Do you see any reason why one might want to keep this?

Reply:
I have removed the lines. I don't think they are required.